omniscidb/QueryEngine/Compiler/Backend.cpp

```diff
@@ -84,9 +84,9 @@ std::shared_ptr<CpuCompilationContext> CPUBackend::generateNativeCPUCode(
   llvm::Module* llvm_module = func->getParent();
   // run optimizations
 #ifndef WITH_JIT_DEBUG
-  llvm::legacy::PassManager pass_manager;
+  llvm::PassBuilder pass_builder;
   compiler::optimize_ir(
-      func, llvm_module, pass_manager, live_funcs, /*is_gpu_smem_used=*/false, co);
+      func, llvm_module, pass_builder, live_funcs, /*is_gpu_smem_used=*/false, co);
```

Comment:
We can probably get rid of the builder parameter, I don't think we prepopulate it anywhere.